src/pages/markets/exchange-traded-funds/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Signup, { Appearances } from 'components/custom/signup'
```

Comment:
please remove this line, we don't need it anymore. thanks :pray:

Reply:
yes removed it